.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow is meant to trigger a build of Docker binaries when a tag
+# is created, it uses the existing `build-docker-image.yml` workflow
+#
+# We use a separate action as we might want to trigger this under
+# different circumstances than a Continuous Deployment, for example.
+#
+# This workflow is triggered if:
+# - A release is published
+# - A pre-release is changed to a release
+name: Release binaries
+
+on:
+  release:
+    types:
+      - released
+
+jobs:
+  # Each time this workflow is executed, a build will be triggered to create a new image
+  # with the corresponding tags using information from git
+  #
+  # The image will be named `zebrad:<semver>`
+  build:
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
I think this can be removed now, as the release type makes sure that we're only publishing releases.

```suggestion
```